api/src/environment.js

```diff
@@ -2,8 +2,13 @@ const url = require('url'),
       logger = require('./logger'),
       { COUCH_URL, UNIT_TEST_ENV } = process.env;
 
+let currentDeployInfo;
+const deployInfo = newDeployInfo => currentDeployInfo = newDeployInfo !== undefined ? newDeployInfo : currentDeployInfo;
```

Comment:
Can we rename this function to `setDeployInfo` or something? Right now it doesn't look like a function to me!

Reply:
Done!